pkg/manage/cmd_set_password.go

```diff
@@ -25,63 +26,84 @@ This command sets the password of a user by a given user id.
 
 // CmdSetPassword initializes the set-password command.
 func CmdSetPassword(cfg *ClientConfig) *cobra.Command {
-	var userID int64
-	var password string
-
 	cmd := &cobra.Command{
 		Use:   "set-password",
 		Short: "Sets an user password.",
 		Long:  setPasswordHelp,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
-			defer cancel()
+	}
 
-			service, close, err := Dial(ctx, cfg.Address)
-			if err != nil {
-				return fmt.Errorf("connecting to gRPC server: %w", err)
-			}
-			defer close()
+	userID := cmd.Flags().Int64P("user_id", "u", 1, "ID of the user account.")
+	password := cmd.Flags().StringP("password", "p", "admin", "New password for the user.")
+	onlyUpdate := cmd.Flags().Bool("update", false, "Do not overwrite an existing password.")
```

Comment:
Hm. The name of this flag is missleading. `$ manage set-password -u 1 -p foo --update` does not mean that the password is updated but that the user keeps its old password.

Maybe change to `keep-existing-password`.

Reply:
I get your point but I don't like long flags. Do you have a shorter alternative?